eng/versioning/version_client.txt

```diff
@@ -12,17 +12,17 @@ com.azure:azure-core-test;1.0.0;1.0.0
 com.azure:azure-data-appconfiguration;1.0.0-preview.6;1.0.0-preview.6
 com.azure:azure-identity;1.0.0;1.0.0
 com.azure:azure-security-keyvault-certificates;4.0.0-preview.5;4.0.0-preview.5
-com.azure:azure-security-keyvault-keys;4.0.0-preview.5;4.0.0-preview.5
-com.azure:azure-security-keyvault-secrets;4.0.0-preview.5;4.0.0-preview.5
+com.azure:azure-security-keyvault-keys;4.0.0;4.0.0
+com.azure:azure-security-keyvault-secrets;4.0.0;4.0.0
 com.azure:azure-messaging-eventhubs;5.0.0-preview.5;5.0.0-preview.5
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.0.0-preview.3;1.0.0-preview.3
 com.azure:azure-sdk-template;1.0.3;1.0.3
-com.azure:azure-storage-blob;12.0.0-preview.5;12.0.0-preview.5
-com.azure:azure-storage-blob-batch;12.0.0-preview.5;12.0.0-preview.5
+com.azure:azure-storage-blob;12.0.0;12.0.0
+com.azure:azure-storage-blob-batch;12.0.0;12.0.0
 com.azure:azure-storage-blob-cryptography;12.0.0-preview.5;12.0.0-preview.5
-com.azure:azure-storage-common;12.0.0-preview.5;12.0.0-preview.5
+com.azure:azure-storage-common;12.0.0;12.0.0
 com.azure:azure-storage-file-share;12.0.0-preview.5;12.0.0-preview.5
```

Comment:
Are we not releasing `azure-storage-file-share` and `azure-storage-blob-cryptography`? @sima-zhu @joshfree

Reply:
+@danieljurek 
This looks like the correct list according to the email yesterday which listed the following:

**Storage (12.0.0)**
- azure-storage-blob
- azure-storage-blob-batch (java and .net only)
- azure-storage-common
- azure-storage-queue

**Keyvault (4.0.0)**
- azure-security-keyvault-keys
- azure-security-keyvault-secrets
This looks like the correct list according to the email.

Reply:
the list of updated package versions is accurate